pages/common/git-reauthor.md

```diff
@@ -0,0 +1,17 @@
+# git reauthor
+
+> Replace details about an author identity, since this command rewrites git's history --force will be needed when `git push` is used.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-reauthor>.
+
+- Change the email and name:
```

Comment:
Does this change all occurences of that author? If so, I'd write
```suggestion
- Change an author's email and name across the whole Git repository:
```

Reply:
Yes